build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -137,15 +137,15 @@ steps:
       yarn smoketest --build "$APP_ROOT/$APP_NAME" --screenshots "$(build.artifactstagingdirectory)/smokeshots" --log "$(build.artifactstagingdirectory)/logs/darwin/smoke.log" --extensionsDir "$(build.sourcesdirectory)/extensions"
     displayName: Run smoke tests (Electron) (Continue on Error)
     continueOnError: true
-    condition: and(succeeded(), and(eq(variables['RUN_TESTS'], 'true'), ne(variables['SMOKE_FAIL_ON_ERROR'], 'true')))
+    condition: and(succeeded(), and(or(eq(variables['RUN_TESTS'], 'true'), eq(variables['RUN_SMOKE_TESTS'], 'true')), ne(variables['SMOKE_FAIL_ON_ERROR'], 'true')))
```

Comment:
If I read this correctly, smoke test will run when RUN_TESTS=true, RUN_SMOKE_TESTS=false. which is not the same as what's in the PR description.

Reply:
```condition: and(succeeded(), and(eq(variables['RUN_SMOKE_TESTS'], 'true'), ne(variables['SMOKE_FAIL_ON_ERROR'], 'true')))```

Reply:
RUN_TESTS = Run all tests (even smoke)
RUN_SMOKE_TESTS = Run ONLY smoke tests

That's the intended behavior (so guess I just didn't do a good enough job of explaining it). 

I did this so we aren't changing the current behavior of RUN_TESTS - at some point maybe we can make it RUN_ALL_TESTS or something but for now I wanted to avoid making too many changes (especially since we share this with VS Code)

Reply:
(updated description for clarity)